zebra-chain/src/proofs/bctv14.rs

```diff
@@ -1,5 +1,8 @@
 use std::{fmt, io};
 
+#[cfg(test)]
+use proptest::{arbitrary::Arbitrary, collection::vec, prelude::*};
+
```

Comment:
nit: I think it would be slightly tidier if these `use` statements were placed below with the `impl Arbitrary` so that they're together with the other `#[cfg(test)]` code.